scripts/license.yaml

```diff
@@ -0,0 +1,21 @@
+permittedLicenses:
+  - MIT
+  - BSD-3-Clause
+  - Apache-2.0
+  - MIT-Modern-Variant
+  - Zlib
+
+rejectedLicenses:
```

Comment:
Yes, the `rejectedLicenses` are "non-functional", except for the colors used when printing the list. 
I thought it would be a good idea to have a list of "Please don't add these to the `permittedLicenses`, because we know they are bad".
I can add a comment to clarify this, or do you think we should just remove `rejectedLicenses`?

Reply:
Well, the list are not exhaustive. That is why I think it gives information, I have added comments that hopefully makes it clear.